create-note/index.js

```diff
@@ -0,0 +1,57 @@
+const functions = require("@google-cloud/functions-framework")
+const axios = require("axios")
+const Tesseract = require("tesseract.js")
+
+const publishMessage = require("./helpers/publish")
+
+functions.http("extractTextHttp", async (req, res) => {
+  await publishMessage("EXTRACT_TEXT_PUBSUB", JSON.stringify(req.body))
+
+  res.status(200).send("OK")
+})
+
+functions.cloudEvent("extractTextPubSub", async (cloudEvent) => {
+  // The Pub/Sub message is passed as the CloudEvent's data payload.
+  const body = JSON.parse(
+    Buffer.from(cloudEvent.data.message.data, "base64").toString()
+  )
+
+  const { pluginInvocationToken, assets, callbackUrl } = body
+
+  console.log("Invoking extract text plugin", pluginInvocationToken)
+
+  try {
```

Comment:
I like having some plain JS examples; I'd suggest that we figure out a consistent structure, so different examples are easier to browse, and I like separating the plugin code (e.g. https://github.com/playbook-labs/pb-plugins/blob/main/split2x2/src/plugin.ts) from the cloud functions runtime code. I'd suggest pulling this part out to a separate file, rather than `publishMessage`